libs/langgraph/langgraph/types.py

```diff
@@ -101,7 +101,10 @@ def default_retry_on(exc: Exception) -> bool:
 
 
 class RetryPolicy(NamedTuple):
-    """Configuration for retrying nodes."""
+    """
```

Comment:
Let's restore the object descriptions.

```suggestion
    """Configuration for retrying nodes.
```